lib/commands/general.js

```diff
@@ -93,5 +93,16 @@ commands.launchApp = iosCommands.general.launchApp;
 
 commands.closeApp = iosCommands.general.closeApp;
 
+commands.keys = async function (keys) {
+  if (!this.isWebContext()) {
+    throw new errors.UnknownError('Command should be proxied to WDA');
+  }
+  let el = await this.active();
```

Comment:
I dont know of any active issue with this. This is something that I came across with our team.